ui/desktop/app/styles/app.scss

```diff
@@ -37,6 +37,11 @@
     overflow: auto;
 
     main {
+      /** This overrides the main defined within Rose.
+       * HDS has this set to 1.5rem top/bottom 3rem left/right. HDS assumes the usage of breadcrumbs
+       * but the DC have no breadcrumbs, so we will keep it at 3rem all around.
```

Comment:
What is meant by `HDS` has set this to `1.5rem`? What's this referring to?

Reply:
@ZedLi Sorry, I might need to re-work that comment and improve my explanation.

HDS with app-frame adds a top/bottom padding of 1.5rem and 3rem on the side. The 1.5rem top padding instead of 3rem is to leave that extra 1.5rem space for the breadcrumbs of the UI's that use this component.
Desktop Client does not use app-frame (but it will in the future). Since currently we do not use the app-frame but want to give a look like we are, we want to add same padding as HDS with app-frame which will be 1.5rem but desktop client does not have breadcrumbs, so instead we define it at 3rem and looks like HDS and gives a feel of consistency within our UI's.

@cameronperera explain it better [within the ticket](https://hashicorp.atlassian.net/browse/ICU-10497).

We tfind out this during connection workflow work and we though it will be a very easy thing to provide consistency until we tackle a refactor to use HDS app-frame. But happy to discuss if you think otherwise.

Reply:
Well stated @calcaide. That is exactly right. This was just something we noticed when working on post connection workflow changes and thought it would move the desktop client closer to matching HDS styling.

Reply:
Do you have a link to these styles? I can't seem to find them

Reply:
The app-frame component docs [are here](https://helios.hashicorp.design/layouts/app-frame).

Also, sorry for the confussion, the padding is applied within `cloud-ui/addons/core` but it within app-frame, here is the code [that adds the padding](https://github.com/hashicorp/cloud-ui/blob/main/addons/core/addon/styles/components/core/layout/main/index.scss#L17)

Reply:
I'm still confused, that looks to be cloud's implementation of their own layout. I also see no recommendation for padding for the app frame HDS component nor any CSS that we need to override in their styles. 

Is this us trying to make it look more like cloud? Is there an actual recommendation from the design team for this change when we say "matching HDS styling"?